CMakeLists.txt

```diff
@@ -61,7 +65,7 @@ set( IAS_PREPRODUCTION_SW 1 CACHE STRING "pre-production mode switch")
 #set( PERFORMANCE_MEASUREMENT 1 CACHE STRING "performance measurement features switch")
 
 # use compiler flags being using in GP1.x:
-SET( CMAKE_CXX_FLAGS " -O2 -pipe -g -fstack-protector-all -pie -fpie -D_FORTIFY_SOURCE=2 -fvisibility-inlines-hidden -DNDEBUG -fexceptions -fstrict-aliasing -Wall -Wextra -Wformat -Wformat-security -Wconversion -Werror -fasynchronous-unwind-tables -fno-omit-frame-pointer -std=c++11" )
+SET( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  -O2 -pipe -g -fstack-protector-all -pie -fpie -D_FORTIFY_SOURCE=2 -fvisibility-inlines-hidden -DNDEBUG -fexceptions -fstrict-aliasing -Wall -Wextra -Wformat -Wformat-security -Wconversion -Werror -fasynchronous-unwind-tables -fno-omit-frame-pointer -std=c++11" )
```

Comment:
This is unrelated to the fix, I presume. Finally, the comment before may not apply, right?